provider/resource_keycloak_openid_client.go

```diff
@@ -377,6 +383,20 @@ func resourceKeycloakOpenidClientCreate(data *schema.ResourceData, meta interfac
 		return err
 	}
 
+	if permissions_enabled, ok := data.GetOkExists("permissions_enabled"); ok {
+		if permissions_enabled.(bool) == true {
+			err := keycloakClient.EnableOpenidClientPermissions(client.RealmId, client.Id)
+			if err != nil {
+				return err
+			}
+		} else {
+			err := keycloakClient.DisableOpenidClientPermissions(client.RealmId, client.Id)
+			if err != nil {
+				return err
+			}
+		}
+	}
```

Comment:
If you agree this would be a part of resourceKeycloakOpenidClientPermissions create method